release-notes/v1_67.md

```diff
@@ -181,6 +181,45 @@ The [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscj
 
 ![Debugger for Java supporting lazy variable resolution during a debugging session](images/1_67/java-lazy-variable.gif)
 
+
+### Jupyter
+
+**Web Extension**
+
+The Jupyter extension can now be installed as a web extension for use at [vscode.dev](https://vscode.dev). Functionality is currently quite limited with support only for notebooks (.ipynb files) being run on Jupyter servers that do not require https.
+
+If you'd like to experiment with the functionality, launch Jupyter from your local machine with:
+```jupyter --no-browser --NotebookApp.allow_origin_pat=https://.*\.vscode-cdn\.net```
+
+And connect to it using the command:
+```Jupyter: Specify Jupyter server for connections```
+
+There are a number of features that will be working on in the coming months:
+- 'https' based servers
+- Intellisense
+- Data viewing
+- Variable viewing
+- Interactive window
+- IPyWidgets
+- Plot viewing
+- Exporting
+
+**Jupyter Powertoys Extension**
+
+We are excited to introduce a new extension for use with Jupyter notebooks that provides experimental functionality for working with them. It includes some new ideas such as:
+- Kernel management panel
+- Contextual help
+- Execution grouping
+
```

Comment:
We'd love for you to take these for a spin and let us know what you think so that we can hone them into a supported feature someday!